src/main/java/de/caritas/cob/userservice/api/admin/service/agency/ConsultantAgencyAdminService.java

```diff
@@ -71,6 +76,52 @@ public ConsultantAgencyResponseDTO findConsultantAgencies(String consultantId) {
         .build();
   }
 
+  public void appendAgenciesForConsultants(Set<ConsultantDTO> consultants) {
+    var consultantIds = consultants.stream()
+        .map(ConsultantDTO::getId)
+        .collect(Collectors.toSet());
+
+    var consultantAgencies = consultantAgencyRepository
```

Comment:
in this case shown deleted consultants on frontend will not have information about agency.
![image](https://user-images.githubusercontent.com/1340555/163835035-480f440c-1779-4553-8ce6-2bc0d77bf126.png)

Reply:
Done